x500/cert/acme/src/test/java/org/wildfly/security/x500/cert/acme/AcmeClientSpiTest.java

```diff
@@ -170,13 +170,27 @@ public void resetServerExpectations() {
     @Test
     public void testCreateAccount() throws Exception {
         final String NEW_ACCT_LOCATION = "http://localhost:4001/acme/acct/384";
-        server = setupTestCreateAccount();
+        server = setupTestCreateAccount(false);
         AcmeAccount account = populateBasicAccount(ACCOUNT_1_V2);
         assertNull(account.getAccountUrl());
         acmeClient.createAccount(account, false);
         assertEquals(NEW_ACCT_LOCATION, account.getAccountUrl());
     }
 
+    @Test
+    public void testCreateAccountMissingLocationURL() throws Exception {
+        server = setupTestCreateAccount(true);
+        AcmeAccount account = populateBasicAccount(ACCOUNT_1_V2);
+        assertNull(account.getAccountUrl());
+
+        try {
+            acmeClient.createAccount(account, false);
+            fail("Expected AcmeException not thrown");
+        } catch(AcmeException e) {
+            assertTrue(e.getMessage().contains("account location URL"));
```

Comment:
Instead of using `contains("account location URL")` here, the account constant could be used instead (i.e., `contains(ACCOUNT)`).